manifests/prepare/package.pp

```diff
@@ -5,21 +5,31 @@
 # @param source
 #   The source file for the puppet-agent package. Can use any of the data types
 #   and protocols that the File resource's source attribute can.
+# @param destination_name
```

Comment:
since the whole class is private because of `assert_private()`, you can just use `# @api private` , then you don't need any further puppet strings docs.

Reply:
I'll fix this in the windows PR